vault/changelog.release.2022-05-17.md

```diff
@@ -0,0 +1,84 @@
+---
+id: o674526ctxg6rt8bc144rdz
+title: 0.95
+desc: ''
+updated: 1652806373844
+created: 1652755065976
+---
+
+Dendron 0.95 has sprouted  🌱
+
+**Local Configuration:** We are releasing an early build of Config Overrides, which allows a local configuration to override values in the default configuration. This lets you use your own settings and vaults within a shared team workspace.
+
+- More information here about [[local configurations|dendron://dendron.dendron-site/dendron.topic.local-config-override]]
+
+**Custom Themes:** You can now customize the style of your sites published with Dendron. Step right up, bring your own custom CSS theme, and be amazed with how good your knowledge can look!
+
+- Learn more about [[custom themes|dendron://dendron.dendron-site/dendron.topic.theme.publish.md]]
+
+**Initialization Speed-Up:** But wait, there's more: less waiting! This release drastically improves workspace initialization speed (10x faster for large workspaces 🚀 ).
+
+## Highlights
+- feat(chore): germ stage implementation of config overrides 
+- feat(publish): Custom theme support for publishing 
+- enhance(workspace): Speed up workspace initialization 
+
+## Everything Else
+- feat(views): allow customization of tree view display and sorting behavior
+- enhance(views): make full graph view as default
+- enhance(views): display workspace information in full graph view
+- enhance(workspace): new user tutorials
+- fix(publish): published images missing leading forward slashes
+
+## Community
+
+### General Updates
+**Self Contained Vaults:**  We hope you're as excited about self contained vaults as we are! To learn more about the newest changes to vaults, check out our [[release notes on self contained vaults|dendron://dendron.blog/blog.2022.2022-05-10-self-contained-vaults-release]]. To see an example of self contained vaults in action, read about how we [use them at Dendron](https://github.com/dendronhq/handbook/blob/main/notes/area.team.sop.use-personal-vault-alongside-team-vaults.md#L8:L8).
```

Comment:
> We hope you're as excited about self contained vaults as we are

This sentence doesn't add much value (no new information given). also, the example is for config overrides, not self contained vaults. can we replace this update with the following?

[[Self contained vaults|dendron://dendron.blog/blog.2022.2022-05-10-self-contained-vaults-release]] make it easier to sync and share your notes with others. They are backwards compatible with existing vaults which means you can experiment with them without breaking your current flow. Starting next week, we will be flipping the defaults so that newly created vaults will be self contained by default. If you would like to preserve the current behavior, you can manually set [[dendron.enableSelfContainedVaultWorkspace|dendron://dendron.dendron-site/dendron.ref.config.vscode-config#dendronenableselfcontainedvaultworkspace]] to `false`.

Reply:
note that the blog link won't resolve so we need to switch to the url